src/sort/sort.rs

```diff
@@ -41,22 +46,105 @@ enum SortMode {
 
 struct Settings {
     mode: SortMode,
+    merge: bool,
     reverse: bool,
     outfile: Option<String>,
     stable: bool,
     unique: bool,
     check: bool,
+    compare_fns: Vec<fn(&String, &String) -> Ordering>,
```

Comment:
This is a decision from before my time. I assume that's because `buf_reader.lines()` is an Iterator<String>, so lines automatically becomes a `Vec<String>`, and `lines.sort_by()` accepts `fn(&String, &String) -> Ordering`